Vector/ViewController/RecentsViewController.m

```diff
@@ -390,6 +390,7 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
             
             MXKRoomDataSourceManager *roomDataSourceManager = [MXKRoomDataSourceManager sharedManagerForMatrixSession:selectedRoomSession];
             MXKRoomDataSource *roomDataSource = [roomDataSourceManager roomDataSourceForRoom:selectedRoomId create:YES];
+            roomDataSource.roomBubbleTableViewCellDelegate = currentRoomViewController;
```

Comment:
Should be removed (see https://github.com/matrix-org/matrix-ios-kit/pull/38 dicussion)